ibis-server/entrypoint.sh

```
@@ -1,5 +1,33 @@
+            --allow-root \
+            --ServerApp.token="" \
+            --ServerApp.password="" \
+            --ServerApp.allow_origin="*" \
+            --ServerApp.base_url="/lab" \
+            --ServerApp.default_url="/lab/tree/notebooks/demo.ipynb" \
+            --ServerApp.iopub_data_rate_limit=1000000000 \
+            --ServerApp.websocket_ping_timeout=30000 \
+            --ServerApp.websocket_ping_interval=30000
+    else
+        # If jupyter with additional arguments, pass all arguments to jupyter
+        echo "Executing: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)